src/pages/ForConsumers.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import NavBar from '../components/NavBar';
+import '../styles/NavBar.css';
+
+//TODO: more to add here. do not change to implicit return
+const ForConsumers = () => {
+    return (
+        <>
+            <NavBar
+                buttons={[]}
+                menuOptions={[
```

Comment:
menuOptions=[{buttonName: 'Log In'}]

Reply:
if you have a login on every page...then dont make it a parameter...bake it into the component

Reply:
i set it up this way because certain pages will have a certain button. others will have no button, so i need the option to put one in. ex: the landing page has a button, the about pages dont.

Reply:
ah i see actually. you are saying if the page does not have a button, just dont add an empty array as a prperty..